packages/web3-eth-ens/test/integration/resolver.test.ts

```diff
@@ -197,6 +198,20 @@ describe('ens', () => {
 		expect(res).toBe(contentHash);
 	});
 
+	// eslint-disable-next-line jest/expect-expect
+	itIf(isSocket)('ContenthasChanged event', async () => {
+		// eslint-disable-next-line no-console, @typescript-eslint/no-misused-promises, no-async-promise-executor
+		await new Promise<void>(async resolve => {
+			const resolver2 = await ens.getResolver('resolver'); // .on('ContenthashChanged', (node, hash) => {});
```

Comment:
What is this comment please (`// .on('ContenthashChanged', (node, hash) => {});`)?